modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

```diff
@@ -99,15 +93,81 @@ Access token URL:: `__<Bitbucket Server URL>__/plugins/servlet/oauth/access-toke
 Authorize URL:: `__<Bitbucket Server URL>__/plugins/servlet/oauth/access-token`
 
 Create incoming link:: Enabled.
+====
 
 .. Fill-in the *Link Applications* form and click the btn:[Continue] button.
-
++
+====
 Consumer Key::  Paste the content of the `__<bitbucket_server_consumer_key>__` file.
 
 Consumer name::  `__<{prod-short}>__`
 
 Public Key:: Paste the content of the `__<public-stripped.pub>__` file.
+====
+
+. Prepare the `secret.yaml` file with specific values as follows:
++
+include::example$snip_bitbucket-personal-access-token-secret.adoc[]
++
+The main parts of the secret are:
++
+[cols=3*]
+|===
+| (1) Label
+| `app.kubernetes.io/component`
+| Indicates it is an SCM personal token secret. 
+
+| (2) Annotation
+| `che.eclipse.org/che-userid`
+| {prod} id of the user token belongs to.
+
+| (3) Annotation
+| `che.eclipse.org/scm-userid`
+| Bitbucket user id to which token belongs.
+
+| (4) Annotation
+| `che.eclipse.org/scm-username`
+| Bitbucket user name to which token belongs.
+
+| (5) Annotation
+| `che.eclipse.org/scm-url`
+| Bitbucket server URL to which this token belongs.
+
+| (6) Annotation
+| `che.eclipse.org/expired-after`
+| Personal access token expiration time (UNIX).
+
+| (7) Data entry
+| `token`
+| Base-64 encoded value of the personal access token. Example:TlRnNE1UazRORFl5TWpNeU9sMTI1aDFFT2dRbXBlTUYvbmhiLzNQUC9hT08=
+|===
++
+NOTE: Encoding a string into the base64 format using the `base64` tool on Linux machines leads to adding the newline character to the end of the source string and causing a value to be unusable as the authentication header value after decoding. Avoid this by using `base64 -w0`, which removes newly added lines, or strip newlines explicitly using`tr -d \\n`.
```

Comment:
Use a proper step, like in  https://www.eclipse.org/che/docs/che-7/end-user-guide/provisioning-the-jetbrains-offline-activation-code/

Reply:
Thats right, but it looks like the step nbr5 and the description+NOTe below it will be removed anyway. Waiting for confirmation now.

Reply:
But maybe I can use it on a different spot .) Thank you, @themr0c